src/components/modals/home/StepOneLogin.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import Button from "../../commons/Button";
+import { ILoginModalStepOne } from "../../../types/types";
+import { Checkbox } from "antd";
+import closeModalHandler from "../../../actions/closeModalHandler";
+import openModalHandler from "../../../actions/openModalHandler";
+import { Modal } from "antd";
+
+const StepOneLogin: React.FC<ILoginModalStepOne> = ({
+  content,
+  label,
+  isOpen,
+  onClose,
+  openModal,
+}) => {
+  const [isChecked, setIsChecked] = useState<boolean>(false);
+
+  const handleCancelChecked = () => {
+    setIsChecked(false);
+  };
```

Comment:
Can we save this function in an utils.ts file? That way we can import it where we need it.

Reply:
solved in commit 424537b8225447b094604e1bc6f94afd41adf270.